include/dfu/dfu_target.h

```diff
@@ -25,7 +25,7 @@ extern "C" {
 struct dfu_target {
 	bool (*identify)(const void * const buf);
 	int (*init)(void);
-	int (*offset)(void);
+	int (*offset)(size_t *offset);
```

Comment:
I think `offset_get` is better, so it's clearer that it gets the offset instead of setting it.

Reply:
renamed